apps/nextjs-website/src/app/page.tsx

```diff
@@ -35,6 +37,13 @@ const Home = async () => {
           svgPath: product.svgPath,
         }))}
       />
+      {nextWebinars.length && (
```

Comment:
```suggestion
      {nextWebinars.length !== 0 && (
```
without that, with the data you are passing right now, you will see a `0` rendered